core/Blitable.carp

```diff
@@ -57,3 +57,7 @@
 (defmodule Uint64
   (defn blit [x] (the Uint64 x))
   (implements blit Uint64.blit))
+
+(defmodule Pointer
```

Comment:
Why is this instance needed?

Reply:
`test/address.carp` won't build unless we have this in place IIRC.